package.json

```diff
@@ -42,8 +42,8 @@
     "jschannel": "^1.0.2",
```

Comment:
Still scroll problems, I think they are all related so I'll stop the review here
When watching [Super group of mine (Thomas)](https://dev.algorea.org/branch/feature/prime-ng-update/en/#/groups/by-id/6710944276987033666;path=52767158366271444,672913018859223173/details) and navigating to suggested activity "Blockly basic task" / Progress / Chapter view:

![image](https://user-images.githubusercontent.com/16257159/153177403-32064c5f-6f95-4ed5-ac5b-edd4b1aaa4b0.png)

Reply:
Duplicat of #901